suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -604,6 +604,18 @@ export const getStakingFiatBalance = (account: Account, rate: number | undefined
     return toFiatCurrency(balanceInEther, rate, 2);
 };
 
+export const getAccountCryptoBalanceWithStakingFormatted = (account: Account) => {
```

Comment:
Please use `calculateBalance` instead of these functions and <CryptoAmountFormatter /> to format it.